patches/net/minecraft/world/level/Level.java.patch

```diff
@@ -5,7 +5,7 @@
  import net.minecraft.world.scores.Scoreboard;
  
 -public abstract class Level implements LevelAccessor, AutoCloseable {
-+public abstract class Level extends net.neoforged.neoforge.common.capabilities.CapabilityProvider<Level> implements LevelAccessor, AutoCloseable, net.neoforged.neoforge.common.extensions.ILevelExtension {
++public abstract class Level implements LevelAccessor, AutoCloseable, net.neoforged.neoforge.common.extensions.ILevelExtension {
```

Comment:
Do we want to keep level attachments or not? They _might_ be useful on the client-side to temporarily hold some data associated with a level

Reply:
Maybe, though that might be unclear to modders that it is just temporary, especially if we are still having attachments removed from the server side in favor of WSD, that still means that people have to manage keeping track of the level (well `ResourceKey<Level>` or whatever) to the data they want for that specific bit. At which point, they can use the same sort of logic without the WSD for the client side.

Reply:
Maybe we remove them for now, and add them back later if people come up with good use cases?

Reply:
Yeah removing for now